src/styles/blocks/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  background: $color-white;
+  border: 1px solid $color-grey;
+  border-radius: 5px;
+  padding: $cards-paddings;
```

Comment:
the same

Reply:
removed  bc prop, and var cards paddings